a10_neutron_lbaas/nova_plumbing_client.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2016, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import appliance_client_base
+import instance_manager
+
+
+class NovaPlumber(object):
```

Comment:
this functionality should be handled by plumbing/network hooks.  if it can't be accommodated, that interface needs enhancing, not a wrapper to the client.